.vscode/settings.json

```
@@ -3,5 +3,9 @@
 		"**/.hemttout": true,
 		"**/include": true,
 		"**/extras": true
-	}
-}
\ No newline at end of file
+	},
+    "files.associations": {
+        "XEH_PREP.hpp": "sqf",
```

```suggestion
        "XEH_PREP.hpp": "sqf",
        "Description.ext": "sqf",
```
Seems that's the capitalization, as [Biki](https://community.bistudio.com/wiki/Description.ext) redirects `description.ext` to `Description.ext` and says `The Description.ext is [...]` at the very start.

^ Then again, the same article (including [the campaign one](https://community.bistudio.com/wiki/Campaign_Description.ext)) uses both capitalizations afterwards, but I'm ignoring that.